zipline/algorithm.py

```diff
@@ -749,9 +749,10 @@ def symbol(self, symbol_str):
         _lookup_date = self._symbol_lookup_date if self._symbol_lookup_date is not None \
             else self.sim_params.period_end
 
-        return self.asset_finder.lookup_symbol_resolve_multiple(
+        return self.asset_finder.lookup_symbol(
             symbol_str,
-            as_of_date=_lookup_date
+            as_of_date=_lookup_date,
+            default_None=False,
```

Comment:
By default, a call to lookup_symbol will return either an Equity or a None if no equity could be mapped. If you set default_None=False, a None will never be returned but, instead, a SymbolNotFound error will be raised.